tests/assets/stn/ensure-passive.html

```diff
@@ -21,6 +21,11 @@
     </script>
     {config}
     {loader}
+    <script>
+      try {
+        document.addEventListener('scroll', function(e){}, {capture:false, passive:true})
```

Comment:
This test was meant to ensure that the listeners the agent sets up are passive.  This test is pointless the way it's constructed if stn does not have the listeners added

Reply:
yes